### PR TITLE
[HOT_FIX] Handle error when user input wrong userid format + fix message test 

### PR DESCRIPTION
**Pull request by tungleu**

### CODE REVIEW

File: src/tests/9_Message.test.ts [hunk 1]
```diff
@@ -7,7 +7,7 @@ import FollowService from '../modules/follows/FollowService'
 import UserService from '../db/service/UserService'
 import UserCollection from '../db/models/user.model'
 
-describe('Message service can', () => {
+describe.only('Message service can', () => {
```
Comment (kennguyen0303) [inline src/tests/9_Message.test.ts:10]: remove .only

**Review by kennguyen0303 — CHANGES_REQUESTED** [review]
remove .only and console.log

**Review by kennguyen0303 — APPROVED** [review]
Approve because it is hot-fix. The fix can be refactored as a middleware (to verify userId in request) and reuse it in multiple route. 

Adding multiple try-catch this time looks really ugly

**Review by Ming424 — APPROVED** [review]
lgtm

**Pull request merged**